Node/src/TeamsChatConnector.ts

```diff
@@ -149,10 +154,18 @@ export class TeamsChatConnector extends builder.ChatConnector {
     this.allowedTenants = null;
   }
 
-  public onQuery(commandId: string, handler: ComposeExtensionQueryHandlerType): void {
+  public onQuery(commandId: string, handler: ComposeExtensionHandlerType): void {
     this.queryHandlers[commandId] = handler;
   }
 
+  public onQuerySettingurl(handler: ComposeExtensionHandlerType) {
+    this.querySettingUrlHandler = handler;
+  }
+
+  public onSetting(handler: ComposeExtensionHandlerType) {
```

Comment:
"onSetting" reads weird, the "XXX" in "onXXX" is typically a verb that indicates when the callback is invoked (BotBuilder appears to stick with this convention: https://github.com/Microsoft/BotBuilder/search?p=3&q=on&type=&utf8=%E2%9C%93).

onSettingsUpdated?